server/api/api/endpoints/jobs.py

```diff
@@ -106,86 +107,3 @@ async def deploy_monitoring_batch_job(
     return {
         "func": batch_function,
     }
-
-
-@router.post("/model-monitoring-controller")
-async def create_model_monitoring_controller(
-    project: str,
-    request: fastapi.Request,
-    auth_info: mlrun.common.schemas.AuthInfo = fastapi.Depends(
-        deps.authenticate_request
-    ),
-    db_session: Session = fastapi.Depends(deps.get_db_session),
-    default_controller_image: str = "mlrun/mlrun",
-    base_period: int = 10,
-):
-    """
-    Submit model monitoring application controller job along with deploying the model monitoring writer function.
-    While the main goal of the controller job is to handle the monitoring processing and triggering applications,
-    the goal of the model monitoring writer function is to write all the monitoring application results to the
-    databases. Note that the default scheduling policy of the controller job is to run every 10 min.
-
-    :param project:                  Project name.
-    :param request:                  fastapi request for the HTTP connection.
-    :param auth_info:                The auth info of the request.
-    :param db_session:               A session that manages the current dialog with the database.
-    :param default_controller_image: The default image of the model monitoring controller job. Note that the writer
-                                     function, which is a real time nuclio functino, will be deployed with the same
-                                     image. By default, the image is mlrun/mlrun.
-    :param base_period:              Minutes to determine the frequency in which the model monitoring controller job
-                                     is running. By default, the base period is 5 minutes.
-    """
-
-    await server.api.utils.auth.verifier.AuthVerifier().query_project_resource_permissions(
-        resource_type=mlrun.common.schemas.AuthorizationResourceTypes.function,
-        project_name=project,
-        resource_name=mlrun.common.schemas.model_monitoring.MonitoringFunctionNames.APPLICATION_CONTROLLER,
-        action=mlrun.common.schemas.AuthorizationAction.store,
-        auth_info=auth_info,
-    )
-
-    if (
-        mlrun.mlconf.httpdb.clusterization.role
-        != mlrun.common.schemas.ClusterizationRole.chief
-    ):
```

Comment:
Since we are no longer submit a scheduled job, we don't need to reroute this API the router. In other words, please remove the `create_model_monitoring_controller` function from `server/api/utils/clients/chief.py` https://github.com/mlrun/mlrun/blob/e0b6e66ccd3524ec130451c19c46be5557ded06b/server/api/utils/clients/chief.py#L215